src/components/hooks/use-auth-check.tsx

```diff
@@ -3,14 +3,16 @@ import { isLoggedIn } from 'common/utility'
 
 const useAuthCheck = () => {
     const [is_logged_in, setLoggedIn] = useState(false)
+    const [is_auth_checked, setIsAuthChecked] = useState(false)
 
     useEffect(() => {
         setLoggedIn(isLoggedIn())
         const checkCookieChange = setInterval(() => setLoggedIn(isLoggedIn()), 800)
+        setIsAuthChecked(true)
```

Comment:
```suggestion
        setIsAuthChecked(true)
        const checkCookieChange = setInterval(() => setLoggedIn(isLoggedIn()), 800)
```

Let's switch these two